pkg/flow/engine.go

```diff
@@ -832,11 +899,12 @@ func (engine *engine) doKnativeHTTPRequest(ctx context.Context,
 		engine.reportError(ar, err)
 		return
 	}
-
 	// add headers
 	req.Header.Add(DirektivDeadlineHeader, deadline.Format(time.RFC3339))
 	req.Header.Add(DirektivNamespaceHeader, ar.Workflow.NamespaceName)
 	req.Header.Add(DirektivActionIDHeader, ar.ActionID)
+	req.Header.Add(DirektivOriginatorIDHeader, ar.Originator)
+	req.Header.Add(DirektivIteratorHeader, fmt.Sprint(ar.Iterator))
```

Comment:
I need to understand the solution better, but my instinct is that we shouldn't have to send this information to the container. That may open us up to exploits.

Reply:
i will check if we still need it